test/sanity/validate-modules/schema.py

```diff
@@ -83,23 +83,51 @@ def return_schema(data):
     )
 
 
+def deprecation_schema():
+
+    deprecation_schema_dict = {
+        # Every time we branch:
+        #   * old versions should be removed from the beginning of this list
+        #   * git rm old modules
+        #   * Update CHANGELOG & porting_guide
+        Required('version'): Any("2.0", "2.2", "2.3", "2.4", "2.5"),
```

Comment:
Should we really enumerate all of these versions here?  Maybe it should be current-5 through current?

Maybe used/calculated like:

```python
from ansible import __version__ as ansible_version
ansible_release_version = float('.'.join(ansible_version.split('.')[:2]))
deprecation_range = [str(round((ansible_release_version-0.4) + float(x/10.0), 2)) for x in range(0, 6)]
Required('version'): Any(*deprecation_range)
```

That would produce something like:

```python
['2.1', '2.2', '2.3', '2.4', '2.5', '2.6']
```

That is basically current-4 through current+1, and could of course be adjusted.